vpos-payment-gateway.php

```diff
@@ -18,6 +18,9 @@
         exit;
     }
 
+    define("VPOS_DIR", plugin_dir_path(__FILE__));
+    error_log(VPOS_DIR);
```

Comment:
Juca, why use error_log when you could use debug or something? Can't you change log level?

Reply:
@kissonde This is in order to log the result to wordpress debug.log file.
than I use `tail -f debug.log` to view logs in my terminal. 

Do you know of an alternative mechanism that could make it easy for me to check results of specific functions during development?